ReactWindows/ReactNative.Shared/UIManager/BorderedCanvas.cs

```diff
@@ -40,5 +38,15 @@ public Border Border
                 Children.Insert(0, _border);
             }
         }
+
+        /// <inheritdoc />
+        protected override AutomationPeer OnCreateAutomationPeer()
+        {
+            return new DynamicAutomationPeer<BorderedCanvas>(this);
+        }
+
+        // TODO: implement runtime change raising event to screen reader #1562
```

Comment:
> TODO: implement runtime change raising event to screen reader #1562 [](start = 10, length = 68)

irrelevant?

Reply:
it's relevant. the setter may need to raise UIA events in some cases, same as for ReactControl.

---
In reply to: [168884352](https://github.com/gmangov/react-native-windows/pull/1#discussion_r168884352) [](ancestors = 168884352)